assets/jsons/generosity_challenge/chat_scripts/chat_script_day_6.json

```diff
@@ -2,11 +2,81 @@
   "main": {
     "type": "textMessage",
     "side": "interlocutor",
-    "text": "Still in progress.. Stay tuned for the new updates!"
+    "text": "Hey {lastName} Family. Your nearly at the end of this challenge!",
+    "next": {
+      "type": "textMessage",
+      "side": "interlocutor",
+      "text": "Our sixth generosity activity is about giving with our money.",
+      "next": {
+        "type": "textMessage",
+        "side": "interlocutor",
+        "text": "To do that I'll need to register your details you've given me so far.",
```

Comment:
I would opt for "the details" instead of "your details" as you are already using "you've given" behind it.

Reply:
I asked Andy to look at the text :) Will wait with a merge/fix till he responds